README.md

```diff
@@ -202,6 +203,32 @@ To send an email, you just need to call the static class:
     Mailer.Send(toAddress, subject, body);
 ```
 
+## Globalization
+
+To globalize our application, we are going to add the localization culture core package to the project:
+
+```bash
+    dotnet add package LocalizationCultureCore --version x.x.x
+```
+
+Then, we configure it on ```ConfigureServices``` method on ```Startup.cs``` by adding this line:
+
+```bash
+    services.AddJsonLocalization(options => options.ResourcesPath = "Resources");
+    services.AddMvc().AddViewLocalization();
+```
+
+We will have to place all the JSON files that the library uses, inside the folder called "Resources" in the project's root directory.
```

Comment:
And locate all the JSON files that the library uses, inside the folder called "Resources" in the project's root directory.